packages/lib/action/service.ts

```diff
@@ -129,6 +129,7 @@ export const createAction = async (data: TActionInput): Promise<TAction> => {
         name,
         type: actionType,
         environmentId,
+        ...(actionType === "code" ? { key: name } : {}),
```

Comment:
This is the functionality we will also remove in the future, right? Since an action shouldn't create an ActionClass automatically.
Can you please add a comment here that this is deprecated with the current date and that it will be removed in the future.

Reply:
We can remove this functionality in this PR itself. I can't think of any edge case here, except the one where if some user from the old version calls the `track` method with an unknown action name, we won't be able to stop that from the js-core package itself (after 2.0.0 we can) but will get an error from the action create API.